sdk/typescript/rooch-sdk/src/session/session.ts

```diff
@@ -138,6 +138,10 @@ export class Session extends Signer {
     return this.localCreateSessionTime
   }
 
+  isExpired() {
```

Comment:
isExpired fun maybe is invalid. After lastActiveTime is updated, the browser does not cache the latest value. Therefore, remove invalid sessions during sdk initialization may not work.

Reply:
Got. After offline discussion, the `isExpired` related logic in rooch-sdk will be temporarily removed.